ramp-database/rampdb/tools/leaderboard.py

```diff
@@ -1,54 +1,59 @@
+from itertools import product
+
+import numpy as np
 import pandas as pd
 
 from ..model.event import Event
 from ..model.event import EventTeam
 from ..model.submission import Submission
 from ..model.team import Team
 
+from .team import get_event_team_by_name
+
 from .submission import get_bagged_scores
 from .submission import get_scores
 from .submission import get_submission_max_ram
 from .submission import get_time
 
+pd.set_option('display.max_colwidth', -1)
```

Comment:
Small note: this will do that session wide, if you only want to do that for a specific call, there is a similar `with pd.option_context(..):` as well